tests/console/openssl_alpn.pm

```diff
@@ -44,4 +44,8 @@ sub run {
     save_screenshot;
 }
 
+sub test_flags {
+    return {milestone => 1};
```

Comment:
What for? It doesn't seem to do any permanent setup?

Reply:
It installs `openssl`, in case of failure the test might rollback to `fips_setup`'s milestone which does not contain `openssl` as http://kepler.suse.cz/tests/6173#step/openssl_fips_hash/4

Reply:
It can fail before or during the installation of `openssl`, in which case it would fail anyway.

IMO it's the responsibility of the `openssl_fips_hash` module that `openssl` is available.